lms/djangoapps/shoppingcart/models.py

```diff
@@ -1746,20 +1746,30 @@ def purchased_callback(self):
         self.course_enrollment.activate()
 
     def additional_instruction_text(self):
+        verification_reminder = ""
+        verified_certificate_text = ""
+        is_enrollment_mode_verified = self.course_enrollment.is_verified_enrollment()  # pylint: disable=E1101
+
+        if is_enrollment_mode_verified:
+            verified_certificate_text = "from the Verified Certificate option "
+
+            domain = microsite.get_value('SITE_NAME', settings.SITE_NAME)
+            path = reverse('verify_student_verify_later', kwargs={'course_id': unicode(self.course_id)})
+            verification_url = "http://{domain}{path}".format(domain=domain, path=path)
+
+            verification_reminder = _(
+                "If you haven't verified your identity yet, please start the verification process ({verification_url})."
+            ).format(verification_url=verification_url)
+
         refund_reminder = _(
-            "You have up to two weeks into the course to unenroll from the Verified Certificate option "
+            "You have up to two weeks into the course to unenroll {verified_certificate_text}"
             "and receive a full refund. To receive your refund, contact {billing_email}. "
             "Please include your order number in your email. "
             "Please do NOT include your credit card information."
-        ).format(billing_email=settings.PAYMENT_SUPPORT_EMAIL)
-
-        domain = microsite.get_value('SITE_NAME', settings.SITE_NAME)
-        path = reverse('verify_student_verify_later', kwargs={'course_id': unicode(self.course_id)})
-        verification_url = "http://{domain}{path}".format(domain=domain, path=path)
-
-        verification_reminder = _(
-            "If you haven't verified your identity yet, please start the verification process ({verification_url})."
-        ).format(verification_url=verification_url)
+        ).format(
+            verified_certificate_text=verified_certificate_text,
+            billing_email=settings.PAYMENT_SUPPORT_EMAIL
+        )
```

Comment:
This seems unnecessarily complicated; I may not have been clear enough with my feedback earlier. I'd like to see this:
```python
verification_reminder = ""
is_enrollment_mode_verified = self.course_enrollment.is_verified_enrollment()  # pylint: disable=E1101

if is_enrollment_mode_verified:
    domain = microsite.get_value('SITE_NAME', settings.SITE_NAME)
    path = reverse('verify_student_verify_later', kwargs={'course_id': unicode(self.course_id)})
    verification_url = "http://{domain}{path}".format(domain=domain, path=path)

    verification_reminder = _(
        "If you haven't verified your identity yet, please start the verification process ({verification_url})."
    ).format(verification_url=verification_url)

refund_reminder = _(
    "You have up to two weeks into the course to unenroll and receive a full refund. "
    "To receive your refund, contact {billing_email}. "
    "Please include your order number in your email. "
    "Please do NOT include your credit card information."
).format(
    verified_certificate_text=verified_certificate_text,
    billing_email=settings.PAYMENT_SUPPORT_EMAIL
)
```